zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to **Apps** in the bottom-right corner.
+
+1. Then, Go to **Admin Panel** -> **Webhooks**.
+
+1. Click on the **Add** button next to **Outgoing Webhook**.
+
+1. Enter the name of the channel for which you want notifications,
+ in **Send messages from a channel** field.
+
+1. Set a name that webhook will post as, in **Name that the webhook will post as** field.
+
+1. Set **Callback URL** to the URL created above.
+
+1. After filling in other details, click **Save Settings**.
```

Comment:
Two things:
* What "other details" are left after doing that?
* The last 4 points should all be one point. :)